config/settings/base.py

```diff
@@ -298,7 +298,7 @@
     "SALESFORCE_ARTICLE_TEXT_INDEX_FIELD", default=False
 )
 SALESFORCE_ARTICLE_LINK_LIMIT = env("SALESFORCE_ARTICLE_LINK_LIMIT", default=100)
-SALESFORCE_API_VERSION = env("SALESFORCE_API_VERSION", default=41)
+SALESFORCE_API_VERSION = env.float("SALESFORCE_API_VERSION", default=41.0)
```

Comment:
Can we just set the default to a string, `default="41.0"`. I don't particularly want to deal with potential floating point rounding errors.

Reply:
I made the change in 120b88 but I wouldn't normally worry about floating point inaccuracies for a number that does not undergo calculations. I think Python round-trips numbers faithfully except for the very small or very large. If you do math on them, all bets are off.